src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.CrossGen.targets

```diff
@@ -41,21 +41,25 @@ Copyright (c) .NET Foundation. All rights reserved.
                                              Condition="'%(CrossgenResolvedAssembliesToPublish.Filename)'=='libclrjit'" />
     </ItemGroup>
 
-    <NETSdkError Condition="'@(_CoreclrResolvedPath->Count())' &gt; 1"
-                 ResourceName="MultipleFilesResolved"
-                 FormatArguments="coreclr" />
-
-    <NETSdkError Condition="'@(_CoreclrResolvedPath)'== ''"
-                 ResourceName="UnableToFindResolvedPath"
-                 FormatArguments="coreclr" />
-
-    <NETSdkError Condition="'@(_JitResolvedPath->Count())' &gt; 1"
-                 ResourceName="MultipleFilesResolved"
-                 FormatArguments="jit" />
-
-    <NETSdkError Condition="'@(_JitResolvedPath)'== ''"
-                 ResourceName="UnableToFindResolvedPath"
-                 FormatArguments="jit" />
+    <NETSdkMessage Condition="'@(_CoreclrResolvedPath->Count())' &gt; 1"
```

Comment:
I think we should have the same task split as built-in tasks NETSdkError, NETSdkWarning, NETSdkMessage. They can share code with a base class. 

I find it nice to read error (being fatal to build) up-front. 

It will also have the nice impact of making this diff easier to review. ;)

Reply:
I like the suggestion. I will make the change.